src/docs/goose-book/src/getting-started/running.md

```diff
@@ -4,87 +4,42 @@ We will use Cargo to run our example load test application. It's best to get in
 
 ```bash
 $ cargo run --release
-   Compiling loadtest v0.1.0 (/home/jandrews/devel/rust/loadtest)
-    Finished dev [optimized] target(s) in 3.56s
+    Finished release [optimized] target(s) in 0.23s
      Running `target/release/loadtest`
 Error: InvalidOption { option: "--host", value: "", detail: "A host must be defined via the --host option, the GooseAttack.set_default() function, or the GooseTaskSet.set_host() function (no host defined for LoadtestTasks)." }
 ```
 
-The load test fails with an error as it hasn't been told the host you want to load test. So, let's try again, this time passing in the `--host` flag. After running for a few seconds, press `ctrl-c` to stop the load test:
+The load test fails with an error as it hasn't been told the host you want to load test. So, let's try again, this time passing in the `--host` flag. We will also add `--report-file` flag, which will generate a HTML report. Same information will also [be printed to the command line](metrics.md) (without graphs). After running for a few seconds, press `ctrl-c` to stop the load test:
```

Comment:
"The same information ...", not "Same information ..." Articles! :)

Reply:
"We will also add the `--report-file`..." also missing a "the"

Reply:
Let's link "will generate a HTML report" to `common.html#writing-an-html-formatted-report`